org.opent2t.onboarding.winkhub/js/common.js

```diff
@@ -8,13 +8,23 @@ class accessTokenInfo {
             accessToken,
             refreshToken,
             tokenType, // ex: 'bearer'
-            scopes // ex: 'full_access'
+            scopes, // ex: 'full_access'
+            expiration // Timestamp for expiration (seconds)
```

Comment:
Unix Standard Time is the number of seconds since the epoch.  [https://en.wikipedia.org/wiki/Unix_time](https://en.wikipedia.org/wiki/Unix_time)

---
In reply to: [96748733](https://github.com/openT2T/onboarding/pull/27#discussion_r96748733) [](ancestors = 96748733)